### PR TITLE
Add function to create W3C Presentation

### PR DESCRIPTION
**Pull request by mi-xu**
Add a function createPresentation that returns a W3C Presentation from a list of W3C VC JWTs

### CODE REVIEW

File: src/Credentials.ts [hunk 1]
```diff
@@ -499,6 +507,21 @@ class Credentials {
     return this.signJWT({ aud, sub, jti, nbf, vc, exp, proof })
   }
 
+  createPresentation({ aud, jti, nbf, exp, vcs }: PresentationParam) {
+    const vp = {
+      '@context': vcs.map(decodeJWT).reduce((allContexts, decodedVc) => {
```
[the remark this answers is not in the document]
Comment (mi-xu) [reply src/Credentials.ts]: I'm not entirely sure.  That seems to be the case to me based on what I can see from the specs and how the test suite was done.  @awoie can you provide any clarification on this?  What values need to be in the `@context` of the presentation?  Does it need to include all the contexts from the vcs it contains?

[the remark this answers is not in the document]
Comment (awoie) [reply src/Credentials.ts]: @mirceanis @mi-xu In the W3C Verifiable Presentation the only mandatory attribute is the `vp` attribute. So, technically no need to have `nbf` in there. Other parameters such as `aud`, `jti` (= optional *id* of the presentation/credential) can be used according to the spec but are not mandatory.

[the remark this answers is not in the document]
Comment (awoie) [reply src/Credentials.ts]: @mi-xu @mirceanis Wrt `@context`, there is no need to fetch the context document from the URL. We implicitly assume that the `@context` is correct. We have to allow users of the lib to define their own context and pass it as a parameter. IMO, it is not feasible to enforce any rules that the provided context is correct except using JSON-LD and resolving the context.

**Comment by mirceanis** [comment]
this functionality is now part of [did-jwt-vc](https://github.com/decentralized-identity/did-jwt-vc)

**Pull request closed without merging**